src/main/java/de/zalando/zmon/dataservice/data/MetricTiers.java

```diff
@@ -0,0 +1,98 @@
+    public static final Logger LOG = LoggerFactory.getLogger(MetricTiers.class);
+
+    private final ObjectMapper objectMapper;
+    private final Executor executor;
+    private final TokenWrapper wrapper;
+    private final String hostname;
+
+    private Integer ingestMaxCheckTier;
+    private Set<Integer> criticalChecks;
+    private Set<Integer> importantChecks;
+
+    public MetricTiers(@DefaultObjectMapper ObjectMapper objectMapper,
+                       @SuppressWarnings("SpringJavaInjectionPointsAutowiringInspection") DataServiceConfigProperties config,
+                       @Qualifier("accessTokens") TokenWrapper wrapper) {
+        this.objectMapper = objectMapper;
+        this.hostname = config.getProxyControllerBaseUrl();
+        this.wrapper = wrapper;
+        this.executor = HttpClientFactory.getExecutor(
+                config.getRestMetricSocketTimeout(),
+                config.getRestMetricTimeout(),
+                config.getRestMetricConnections(),
+                config.getConnectionsTimeToLive()
+        );
+    }
+
+    @Scheduled(fixedRate = 60_000)
+    public void updateTiers() {
+        LOG.debug("Updating metric tiers configuration");
+        try {
+            this.ingestMaxCheckTier = getMaxCheckTier();
+
+            final Map<String, Set<Integer>> checkTiers = getCheckTiers();
+            this.criticalChecks = checkTiers.get("critical");
+            this.importantChecks = checkTiers.get("important");
+        } catch (IOException e) {
+            LOG.error("Metric tiers configuration update failed", e);
+        }
+    }
+
+    public boolean isMetricEnabled(int checkId) {
+        if (ingestMaxCheckTier <= 0 || ingestMaxCheckTier >= 3) return true;
+        else if (ingestMaxCheckTier == 1) return criticalChecks.contains(checkId);
+        else return criticalChecks.contains(checkId) || importantChecks.contains(checkId);
+    }
+
+    private Integer getMaxCheckTier() throws IOException {
+        final Optional<JsonNode> data = getEntityData("zmon-service-level-config");
+        return data.map(d -> d.get("ingest_max_check_tier")).map(JsonNode::asInt).orElse(0);
+    }
+
+    private Map<String, Set<Integer>> getCheckTiers() throws IOException {
+        final Optional<JsonNode> data = getEntityData("zmon-check-tiers");
+
+        final Set<Integer> criticalChecks = getCheckList(data, "critical");
+        final Set<Integer> importantChecks = getCheckList(data, "important");
+
+        return ImmutableMap.of("critical", criticalChecks, "important", importantChecks);
+    }
+
+    private Optional<JsonNode> getEntityData(final String entityId) throws IOException {
+        final String uri = hostname + "/api/v1/entities/" + entityId;
+        final Request request = Request.Get(uri).addHeader("Authorization", "Bearer " + wrapper);
+        final String response = executor.execute(request).returnContent().toString();
+        final JsonNode jsonNode = objectMapper.readTree(response);
```

Comment:
What do you think about writing a `DTO` class for the entity structure and use `mapper.readValue(response, CheckTiersEntity.class)`?
Like here: https://github.com/zalando-zmon/zmon-controller/blob/a0ea5cae6384318e5763ca699c8133bbf9b60b6e/zmon-controller-app/src/main/java/org/zalando/zmon/api/domain/entity/specific/CheckTiersEntity.java

It would decouple representation of how it stored from logic.